pipwin/pipwin.py

```diff
@@ -311,8 +312,7 @@ def install(self, requirement):
         Install a package
         """
         wheel_file = self.download(requirement)
-        pip._internal.main(["install", wheel_file])
-
+        subprocess.check_call([sys.executable, '-m', 'pip', 'install',  wheel_file])
```

Comment:
not working as `sys` is not in scope. can you fix that? also can you also change the uninstall command to use subprocess as `_internal` is not working any more